src/Annotations/AppKernel.php

```diff
@@ -0,0 +1,292 @@
+            $notifyingHttpClient = new NotifyingHttpClient(
+                new BatchingHttpClient(
+                    new Guzzle6HttpClient(
+                        $app['guzzle']
+                    ),
+                    $app['api.requests_batch']
+                )
+            );
+            if ($app['debug']) {
+                $logger = $app['logger'];
+                $notifyingHttpClient->addRequestListener(function ($request) use ($logger) {
+                    $logger->debug("Request performed in NotifyingHttpClient: {$request->getUri()}");
+                });
+            }
+
+            return new ApiSdk($notifyingHttpClient);
+        };
+
+        $this->app['aws.sqs'] = function (Application $app) {
+            $config = [
+                'version' => '2012-11-05',
+                'region' => $app['aws']['region'],
+            ];
+            if (isset($app['aws']['endpoint'])) {
+                $config['endpoint'] = $app['aws']['endpoint'];
+            }
+            if (!isset($app['aws']['credential_file']) || $app['aws']['credential_file'] === false) {
+                $config['credentials'] = [
+                    'key' => $app['aws']['key'],
+                    'secret' => $app['aws']['secret'],
+                ];
+            }
+
+            return new SqsClient($config);
+        };
+
+        $this->app['aws.queue'] = function (Application $app) {
+            return new SqsWatchableQueue($app['aws.sqs'], $app['aws']['queue_name']);
+        };
+
+        $this->app['aws.queue_transformer'] = function (Application $app) {
+            return new SqsMessageTransformer($app['api.sdk']);
+        };
+
+        $this->app->register(new ConsoleServiceProvider(), [
+            'console.name' => 'Annotations console',
+            'console.version' => '0.1.0',
+            'console.project_directory' => __DIR__.'/../..',
+        ]);
+
+        $this->app->register(new QueueCommandsProvider(), [
+            'sqs.queue_message_type' => $this->app['aws']['queue_message_default_type'],
+            'sqs.queue_name' => $this->app['aws']['queue_name'],
+            'sqs.region' => $this->app['aws']['region'],
+        ]);
+
+        // Cache.
+        if ($this->app['ttl'] > 0) {
+            $this->app->after([$this, 'cache'], 3);
+        }
+    }
+
+    public function handle(Request $request, $type = self::MASTER_REQUEST, $catch = true) : Response
+    {
+        return $this->app->handle($request, $type, $catch);
+    }
+
+    public function terminate(Request $request, Response $response)
+    {
+        $this->app->terminate($request, $response);
+    }
+
+    public function get($id)
+    {
+        if (!isset($this->app[$id])) {
+            throw new UnknownIdentifierException($id);
+        }
+
+        return $this->app[$id];
+    }
+
+    public function has($id) : bool
+    {
+        return isset($this->app[$id]);
+    }
+
+    public function getApp() : Application
```

Comment:
This doesn't look needed. Only appears to be used in tests, where `AppKernel` itself can be used.

Reply:
I'll remove.